infra/terraform/test-org/ci-triggers/locals.tf

```diff
@@ -24,6 +24,7 @@ locals {
   ]
   exclude_repos = [
     "ci-example-foundation",
+    "ci-cloud-foundation-training"
```

Comment:
This actually fails as it tried to link to repo `terraform-cloud-foundation-training` and repo name is `cloud-foundation-training`. Should I add another exception for this or create 2 individual triggers it in [triggers.tf](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/b6e7c413bf0a817d07899a15104343f1fe3ab1b3/infra/terraform/test-org/ci-triggers/triggers.tf)?

Reply:
Override is good.